tests/integration/calendar/test_date_picker_range.py

```diff
@@ -109,3 +109,39 @@ def test_dtpr004_max_and_min_dates_are_clickable(dash_dcc):
         '#dps-initial-month .DateInput_input.DateInput_input_1[placeholder="End Date"]',
         "01/20/2021",
     )
+
+
+def test_dtpr005_disabled_days_arent_clickable(dash_dcc):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(
+        [
+            html.Label("Operating Date"),
+            dcc.DatePickerRange(
+                id="dpr",
+                min_date_allowed=datetime(2021, 1, 1),
+                max_date_allowed=datetime(2021, 1, 31),
+                initial_visible_month=datetime(2021, 1, 1),
+                disabled_days=[datetime(2021, 1, 10), datetime(2021, 1, 11)],
+            ),
+        ],
+        style={
+            "width": "10%",
+            "display": "inline-block",
+            "marginLeft": 10,
+            "marginRight": 10,
+            "marginBottom": 10,
+        },
+    )
+    dash_dcc.start_server(app)
+    date = dash_dcc.find_element("#dpr input")
+    assert not date.get_attribute("value")
+    assert not any(
+        dash_dcc.select_date_range("dpr", day_range=(10, 11))
+    ), "Disabled days should not be clickable"
+    assert all(
+        dash_dcc.select_date_range("dpr", day_range=(1, 2))
+    ), "Other days should be clickable"
+
+    # open datepicker to take snapshot
+    date.click()
+    dash_dcc.percy_snapshot("dtpr005 - disabled days")
```

Comment:
@alexcjohnson please check this definitely takes a snapshot with the datepicker open. I think it does, but wasn't quite sure how to check locally?

Reply:
All good, once I fixed the other tests and Percy ran I was able to see them myself.